### PR TITLE
Add ContentSecuritPolicy flag

### PR DESCRIPTION
**Pull request by taylorsilva**
# Changes proposed in this pull request
Add the CSP flag from concourse/concourse#6949

# Contributor Checklist
- [x] Variables are documented in the `README.md`
- [x] Which branch are you merging into?
    - `master` is for changes related to the current release of the `concourse/concourse:latest` image and should be good to publish immediately
    - `dev` is for changes related to the next release of Concourse (aka unpublished code on `master` in [concourse/concourse](https://github.com/concourse/concourse))

# Reviewer Checklist
> This section is intended for the core maintainers only, to track review progress. Please do not
> fill out this section.
- [ ] Code reviewed
- [ ] Topgun tests run
- [ ] Back-port if needed
- [ ] Is the correct branch targeted? (`master` or `dev`)

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -836,10 +836,16 @@ concourse:
...
-    ## The value to set for X-Frame-Options. If omitted, the header is not set.
+    ## The value to set for X-Frame-Options. If omitted, the default value in
+    ## the Concourse binary is used.
     ##
     xFrameOptions:
 
+    ## The value to set for Content-Security-Policy header. If omitted, the default
+    ## value in the Concourse binary is used.
+    ##
+    ContentSecuritPolicy:
```
Comment (aoldershaw) [inline values.yaml:847]:
```suggestion
    ContentSecurityPolicy:
```

Also, is it uncommon for the value to be capitalized? Seems like most follow `lowerCamelCase`

Comment (taylorsilva) [reply values.yaml]: Can you tell I didn't get enough sleep? 😅

File: templates/web-deployment.yaml [hunk 2]
```diff
@@ -754,9 +754,9 @@ spec:
             - name: CONCOURSE_X_FRAME_OPTIONS
               value: {{ .Values.concourse.web.xFrameOptions | quote }}
             {{- end }}
-            {{- if .Values.concourse.web.ContentSecuritPolicy }}
+            {{- if .Values.concourse.web.contentSecuritPolicy }}
             - name: CONCOURSE_CONTENT_SECURITY_POLICY
-              value: {{ .Values.concourse.web.ContentSecuritPolicy | quote }}
+              value: {{ .Values.concourse.web.contentSecuritPolicy | quote }}
```
Comment (aoldershaw) [inline templates/web-deployment.yaml:759]: Looks like this (and above) still has the `Securit` typo (missing the "y")

Comment (taylorsilva) [reply templates/web-deployment.yaml:759]: CAN YOU TELL I NEED MORE SLEEP 🙃

Comment (taylorsilva) [reply templates/web-deployment.yaml]: This time I used `ag` to make sure I caught all my typos!

**Review by aoldershaw — APPROVED** [review]
LGTM!

**Pull request merged**